examples/multiplatform-chat/src/clientJsMain/kotlin/App.kt

```diff
@@ -17,7 +17,7 @@
 import kotlinx.coroutines.flow.*
 
 suspend fun main() {
-    val api = connectToApi("Yuri")
+    val api = connectToApiUsingWS("Yuri")
```

Comment:
Thanks... I guess.

Reply:
hah, yes, just first name which comes in mind after mine :)